disnake/role.py

```diff
@@ -366,6 +366,26 @@ def is_assignable(self) -> bool:
             and (me.top_role > self or me.id == self.guild.owner_id)
         )
 
+    def has_gradient(self) -> bool:
+        """
+        Whether the role has any gradient (custom or holographic).
+
+        .. versionadded:: 2.12
+
+        :return type: :class:`bool`
+        """
+        return self.secondary_color is not None
+    
+    def is_holographic(self) -> bool:
```

Comment:
thoughts on naming this `has_holographic_style `? @Enegg  @onerandomusername  (suggested by vi)

Reply:
How about `has_color_gradient` and `has_holographic_color` then, respectively